lib/vanagon/component.rb

```diff
@@ -12,7 +12,7 @@ class Component
     attr_accessor :settings, :platform, :patches, :requires, :service, :options
     attr_accessor :directories, :replaces, :provides, :cleanup_source
     attr_accessor :sources, :preinstall_actions, :postinstall_actions
-    attr_accessor :preremove_actions, :postremove_actions, :license
+    attr_accessor :preremove_actions, :postremove_actions, :license, :upstream_url
```

Comment:
nitpick, but I'd like to see the `:upstream_url` alongside the `:url`. I like logical groupings.